index.html

```diff
@@ -399,7 +399,7 @@ <h3>What other people think</h3>
                                         Marcus Hoffmann
                                     </a>
                                 </p>
-                                <p class="user-statement">one of the most used apps on my Phone</p>
+                                <p class="user-statement">one of the most used apps on my phone</p>
```

Comment:
Shouldn't user statements be kept as is as they are quoted?

Reply:
True, but there should also be the correct quote signs, and they shouldn't be red. Leaving for now.

Reply:
you could write a `/sic` if you want to

Reply:
It wans't in the original quote

Reply:
`sic` is used when you're quoting someone but you want to indicate that whatever error the reader sees is part of the quote and not a typo caused by you.

Reply:
It ostensibly isn't _important_ to Marcus to be wrong about how to write "phone"…
Nor is it some level of detail the reader should be privy to. Fixing errors suffices as a solution.